R/tidyverse.R

```diff
@@ -85,13 +85,15 @@ create_tidy_package <- function(path, copyright_holder = NULL) {
 
   use_cran_comments(open = FALSE)
 
-  ui_todo("In the new package, remember to do:")
-  ui_todo("{ui_code('use_git()')}")
-  ui_todo("{ui_code('use_github()')}")
-  ui_todo("{ui_code('use_tidy_github()')}")
-  ui_todo("{ui_code('use_tidy_github_actions()')}")
-  ui_todo("{ui_code('use_tidy_github_labels()')}")
-  ui_todo("{ui_code('use_pkgdown_github_pages()')}")
+  ui_bullets(c(
+    "i" = "In the new package, remember to do:",
+    "_" = "{.run usethis::use_git()}",
+    "_" = "{.run usethis::use_github()}",
+    "_" = "{.run usethis::use_tidy_github()}",
+    "_" = "{.run usethis::use_tidy_github_actions()}",
+    "_" = "{.run usethis::use_tidy_github_labels()}",
+    "_" = "{.run usethis::use_pkgdown_github_pages()}"
+  ))
```

Comment:
I think a better approach here would be to have `ui_code_snippet()` copied to clipboard that will be pastable in the new project. These runnable hyperlinks will be available for the current session, not the new one with the new package.

Reply:
If you'd like to make PR with these ideas after the merge, let's do that.